ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -87,31 +124,44 @@ class CellEditorOverlay extends Component {
   }
 
   handleSaveCell() {
-    const {queriesWorkingDraft, cellWorkingType, cellWorkingName} = this.state
+    const {
+      queriesWorkingDraft,
+      cellWorkingType: type,
+      cellWorkingName: name,
+      axes,
+    } = this.state
+
     const {cell} = this.props
 
-    const newCell = _.cloneDeep(cell)
-    newCell.name = cellWorkingName
-    newCell.type = cellWorkingType
-    newCell.queries = queriesWorkingDraft.map(q => {
+    const queries = queriesWorkingDraft.map(q => {
```

Comment:
The new logic of `axes` throughout the various components should handle the lack of label on queries from here on out.